demos/apps/bookinfo/README.md

```diff
@@ -348,10 +340,17 @@ If you now refresh the `productpage` you'll see that while the load generator is
    $ ./cleanup.sh
    ```
 
+1. Shutdown Istio control plane (optional)
+
+   ```bash
+   $ kubectl delete -f ../../../kubernetes/istio-install
+
+   ```
+
```

Comment:
if you want to add cleanup instructions in the istio-install, I can change this step to refer to those